css/CSS2/fonts/font-family-rule-004.xht

```diff
@@ -5,7 +5,6 @@
         <link rel="author" title="Microsoft" href="http://www.microsoft.com/" />
         <link rel="help" href="http://www.w3.org/TR/CSS21/fonts.html#font-family-prop" />
         <link rel="help" href="http://www.w3.org/TR/css-fonts-3/#font-family-prop"/>
-        <meta name="flags" content="font" />
         <meta name="assert" content="Font family named 'inherit' needs to be quoted to avoid conflict with keyword 'inherit'." />
```

Comment:
This is a [visual test](https://web-platform-tests.org/writing-tests/visual.html) and isn't run in WPT. There's a PREREQUISITE bit below that points to a font that doesn't exist. But that's fine, it's not more broken after this change.